### PR TITLE
[SW-761] Fix unnecessary fetching of latest transactions

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

File: src/popup/components/LatestTransactionsCard.vue [hunk 1]
```diff
@@ -113,21 +112,28 @@ export default defineComponent({
...
-    });
+    }
+
+    // To avoid unnecessary data transfers instead of constant polling
+    // we are fetching the transactions only if the total balance of the accounts changes.
+    watch(
+      balancesTotal,
+      (val, oldVal) => {
+        if (!latestTransactions.value.length || val !== oldVal) {
+          setTimeout(() => updateData(), 100);
```
Comment (CedrikNikita) [inline src/popup/components/LatestTransactionsCard.vue:125]: Transaction appearing on the middleware slower than node updates the balance. So the latest transaction will not be fetched in most of cases.

File: src/popup/components/LatestTransactionsCard.vue [hunk 2]
```diff
@@ -113,21 +112,28 @@ export default defineComponent({
...
-    watch(balances, updateData);
 
-    onMounted(async () => {
-      await updateData();
       isLoading.value = false;
-    });
+    }
+
+    // To avoid unnecessary data transfers instead of constant polling
+    // we are fetching the transactions only if the total balance of the accounts changes.
```
Comment (CedrikNikita) [inline src/popup/components/LatestTransactionsCard.vue:120]: We need also to watch changes of token balances for every account, in order to cover also fungible token transactions.

**Review by CedrikNikita — CHANGES_REQUESTED** [review]

File: src/popup/components/LatestTransactionsCard.vue (hunk 2 above)
Comment (CedrikNikita) [reply src/popup/components/LatestTransactionsCard.vue:120]: #1928

**Review by pstachel — APPROVED** [review]

File: src/popup/components/LatestTransactionsCard.vue (hunk 2 above)
Comment (peronczyk) [reply src/popup/components/LatestTransactionsCard.vue:120]: Your PR is fixing this: https://github.com/aeternity/superhero-wallet/pull/1928/files
We'll merge both of them in order to fix the whole problem.

File: src/popup/components/LatestTransactionsCard.vue (hunk 1 above)
Comment (peronczyk) [reply src/popup/components/LatestTransactionsCard.vue]: As discussed on priv I increased the delay. In the future we need to replace this with websockets.

**Review by CedrikNikita — APPROVED** [review]

**Pull request merged**